common/network-common/src/test/java/org/apache/spark/network/client/TransportClientFactorySuite.java

```diff
@@ -45,6 +41,8 @@
 import org.apache.spark.network.util.JavaUtils;
 import org.apache.spark.network.util.TransportConf;
 
+import static org.junit.Assert.*;
```

Comment:
Maybe we should avoid import `*`?

Reply:
`org.junit.Assert.*` seems wildly used in spark

Reply:
got it.